static/templates/settings_tab.handlebars

```diff
@@ -114,6 +114,15 @@
             {{t "Display time in 24-hour format (17:00 instead of 5:00 PM)" }}
           </label>
         </div>
+
+        <div class="control-group">
+          <label for="default_language" class="control-label">{{t "Default Language" }}</label>
+          <select name="default_language" id="default_language" value="{{ page_params.default_language }}">
+            <option selected="selected" value="en"> {{t "English" }}</option>
+            <option value="zh_CN"> {{t "Simplified Chinese" }}</option>
```

Comment:
I wonder whether we want to display the names of the other languages in that language?  

Regardless, maybe we should get the list of supported languages from the backend?